okhttp/src/main/java/okhttp3/ConnectionSpec.java

```diff
@@ -40,6 +40,17 @@
  */
 public final class ConnectionSpec {
 
+  // Most secure but supported list as defined by Google Cloud Platform.
```

Comment:
This list also shows up in the HTTP/2 spec (or its drafts?)
I'd rather not discuss Google Cloud here because I don't want to treat Google’s selection as a specification.